test/lightning/pipeline/state_assembler_test.exs

```diff
@@ -74,7 +74,7 @@ defmodule Lightning.Pipeline.StateAssemblerTest do
 
       assert StateAssembler.assemble(run) |> Jason.decode!() == %{
                "configuration" => %{"other" => "credential"},
-               "error" => ["I've failed, log log"],
+               "error" => "I've failed, log log",
```

Comment:
The error value in job state is expected to be an array.

Reply:
Reverted